src/resources/Sources/SourcesSubInterfaces/Permission.ts

```diff
@@ -40,6 +40,57 @@ export interface DocumentPermissionModel {
     state?: DocumentPermissionState;
 }
 
+export interface SecurityIdentityInfoModel {
+    key?: string;
+    value?: string;
+}
+
+export interface DetailedSecurityIdentityModel {
+    name?: string;
+    provider?: string;
+    type?: string;
+    infos?: SecurityIdentityInfoModel[];
+    declaration?: string;
```

Comment:
What is this property for?

Reply:
Looks like it's an `XML` representation of the `name`, `type` and `infos` fields (based on what I could find in the code). Here's an example. Not sure what's the use of this though.

<img width="905" alt="Screenshot 2023-07-17 at 10 02 49 AM" src="https://github.com/coveo/platform-client/assets/133259861/2c902146-a031-4796-a16e-4cc3741cf39b">

Reply:
alright, then I'm fine leaving it as-is.